logstash-core/lib/logstash/compiler/lscl/lscl_grammar.rb

```diff
@@ -1,21 +1,7 @@
-# Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
It seems we shouldn't remove license note.

Reply:
yeah - not sure how it got there, 
given this file is auto-generated and the source does not have the license